old-tests/webdriver/README.md

```diff
@@ -12,12 +12,12 @@ whether they meet the recognized standard.
 3. Run the test against a different config specified in webdriver.cfg:
    `WD_BROWSER=chrome python runtests.py`
 
-To be run a specific test file you can just run `python test_file.py`
+To run a specific test file you can just run `python test_file.py`
```

Comment:
Done, but for some reason the final line still shows as different. I ran the 'rich diff' and it said there was nothing to show. I'm not sure how to fix that.